third_party/nvfuser/csrc/python_frontend/fusion_record.h

```diff
@@ -1099,12 +1101,42 @@ struct TensorRecord : RecordFunctor {
   }
 
   virtual void operator()(FusionState& fd) final {
-    auto tv = TensorViewBuilder()
-                  .ndims(symbolic_sizes_.size())
-                  .contiguity(contiguous_info_)
-                  .shape(symbolic_sizes_)
-                  .dtype(dtype_)
-                  .build();
+    // auto tv = TensorViewBuilder()
+    //               .ndims(symbolic_sizes_.size())
+    //               .contiguity(contiguous_info_)
+    //               .shape(symbolic_sizes_)
+    //               .dtype(dtype_)
+    //               .build();
+    std::vector<IterDomain*> sizes;
+    std::vector<bool> contig_info;
+    int rank = symbolic_sizes_.size();
+
+    for (const auto i : c10::irange(rank)) {
+      int index = i;
```

Comment:
nit: Why not
```suggestion
    for (const auto index : c10::irange(rank)) {
```

Reply:
Hahaha, sorry, this is from debugging where I though I need to push_back in the reverse order.